lib/config.js

```diff
@@ -65,6 +65,15 @@ const Config = {
       default: true,
       order: 4
     },
+    wrapOutput: {
+      title: "Enable Soft Wrap for Output",
+      includeTitle: false,
+      description:
+        "If enabled, your output code from hydrogen will break long text and items.",
```

Comment:
This is very small thing, but `hydrogen` would be better to be camel-case for consistency with other config descriptions: `Hydrogen`

Reply:
Agreed.